suse_migration_services/units/migrate.py

```diff
@@ -84,9 +84,15 @@ def get_migration_product():
     zypper migration call
     """
     config = _read_migration_config()
-    return config['migration']['target']['product']
+    migration_product = config['migration']['target']['product']
```

Comment:
I thought we had agreed not to layer this and just use:

```
migration_product: $VALUE
```
rather than

```
migration:
  - target:
      - product: $VALUE
```

Reply:
@rjschwei yes, this PR is old and [1](https://github.com/SUSE/suse-migration-services/pull/81) ahs the right structure.
I can update this code once [1] PR is merged to use that code and structure.